### ISSUE TITLE
Service Names that contain dots create error on deployment

### ISSUE THREAD

**Issue by eliasnawfal** [Azure/service-fabric-issues#213]
If I have 2 projects with names M.Project.Website and A.Project.Website will create Parameters having same name in ApplicationManifest.xml
```
    <Parameter Name="Website_InstanceCount" DefaultValue="-1" />
    <Parameter Name="Website_InstanceCount" DefaultValue="-1" />
```
The parameter takes what is after last dot from project name
This will lead to an error Website_InstanceCount exists more than once

**Comment by dbreshears** [comment]
Thanks for the feedback.   We will investigate a fix on the tooling side to make sure these are unique

**Comment by dbreshears** [comment]
I'll close this as I created a bug in our internal  database

**Comment by wasabii** [comment]
Is there any word on this? This is insane that you basically cannot build SF services in Visual Studio right now unless you hack all your project names up. This is from March?

**Comment by dbreshears** [comment]
@wasabii.   A fix for this in progress.  I was under the impression this mainly created duplicate parameters and could easily fix them as a workaround, but seems it may be more than that if you are having to hack up project names and having build issues.   Can you share more info on the issues you are hitting and what service projects you are using.

**Comment by dbreshears** [comment]
Closing. This should be fixed in the tooling for the 5.7 release.
